src/plots/gl3d/layout/axis_attributes.js

```diff
@@ -14,6 +14,7 @@ var extendFlat = require('../../../lib/extend').extendFlat;
 
 
 module.exports = {
+    visible: axesAttrs.visible,
```

Comment:
`visible` for 3D axes doesn't actually works, correct?

I think you _had_ to put it in there so that `supplyDefaults` doesn't error out.